releasenotes/notes/add-leap-api-endpoint-config-option-28607cc0cb2e360d.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Add config options for defining Leap API endpoint: ``leap_api_endpoint``
```

Comment:
config --> configuration
(At least here where it's formal customer documentation)